File: Angular4_03.05/Server/Models/Lot.cs
using System.ComponentModel.DataAnnotations;

namespace Server.Models
{

public class Lot{
    public string OwnerCurrency {get;set;} = default!;
    public string RequiredCurrency {get;set;} = default!;
    public string OwnerName {get;set;} = default!;
    public double Price {get;set;}=default!;
    

}
}

File: Azure4_03.05/currency-lots/src/Lot.ts
 enum Currency{
    PLN = "PLN",
    USD="USD",
    UAH="UAH",
    RUB="RUB",
    EUR="EUR",
    CAD="CAD",
    CNY="CNY",
    SEK="CNY"
}
 class Lot{
    constructor(public OwnerCurrency: Currency,public RequiredCurrency:Currency, public Price:number, public OwnerName:string){}
}
type Currencies = keyof Currency;
export {Currency,Lot,Currencies};